gmd.py

```
@@ -322,6 +542,611 @@ def interactive_menu(env: CommandEnv) -> None:
+            for issue in report.issues:
+                table.add_row(issue.provider, issue.severity, str(issue.path), issue.message)
+            ui.console.print(table)
+        except Exception:
+            pass
+    lines.append(f"Found {len(report.issues)} issue(s):")
+    for issue in report.issues:
+        lines.append(f"- [{issue.severity}] {issue.provider}: {issue.path} — {issue.message}")
+    ui.summary("Doctor", lines)
+
+
+def run_stats_cli(args: argparse.Namespace, env: CommandEnv) -> None:
```

Ruff (0.15.9)

[warning] Too many branches (17 > 12)

(PLR0912)